docs/sources/next/results-output/real-time/opentelemetry.md

```diff
@@ -1,12 +1,14 @@
 ---
 title: 'OpenTelemetry'
-description: 'Use the OpenTelemetry output extension to send test results in OTEL metrics format.'
+description: 'Use the OpenTelemetry experimental output to send test results in OTEL metrics format.'
 weight: 00
 ---
 
 # OpenTelemetry
 
-Grafana k6 can push test run metrics in the [OpenTelemetry (OTEL) metrics format](https://opentelemetry.io/docs/specs/otel/metrics/) to an OTEL metrics collector or a metrics backend that supports the OTEL metrics format by using the [xk6-output-opentelemetry extension](https://github.com/grafana/xk6-output-opentelemetry/).
+Grafana k6 can push test run metrics in the [OpenTelemetry (OTEL) metrics format](https://opentelemetry.io/docs/specs/otel/metrics/) to an OTEL metrics collector or a metrics backend that supports the OTEL metrics format by using the experimental OpenTelemetry output `--out experimental-opentelemetry`.
+
+Note that the `experimental-opentelemetry` became a part of k6 only with k6 v0.53.0, so prior that version you should use the [xk6-output-opentelemetry extension](https://github.com/grafana/xk6-output-opentelemetry/).
 
 {{< admonition type="note" >}}
```

Comment:
Shouldn't we also update this note? It states: `This is an extension, and we're looking for feedback and contributions and aiming to graduate it as an experimental output within Grafana k6 in a future release.`, but I think that time arrived, or is about to! 😛

Reply:
Good catch! Missed that!